showcase/src/Dialog.elm

```diff
@@ -18,34 +17,79 @@ stories : RenderConfig -> ExplorerUI
 stories cfg =
     storiesOf
         "Dialog"
-        [ dialog cfg
+        [ dialogDesktop cfg
+        , dialogMobile
         ]
 
 
-dialog : RenderConfig -> ExplorerStory
-dialog cfg =
+dialogDesktop : RenderConfig -> ExplorerStory
+dialogDesktop cfg =
     story
-        ( "Dialog"
+        ( "Dialog Desktop"
         , Element.column
             [ Element.centerX
             , Element.centerY
             , Element.padding 20
             , Element.height <| Element.px 500
-            , overlayBackground
+            , Element.width Element.fill
             ]
             [ iconsSvgSprite
-            , Dialog.dialog "Dialog title"
-                (Icon.warning "Warning dialog")
-                Msg.NoOp
+            , Dialog.dialog demoText.title
+                (Icon.warning demoText.icon)
                 |> Dialog.withBody
-                    ("Dialog body text" |> Text.body2 |> Text.renderElement cfg)
+                    (demoText.body |> Text.body2 |> Text.renderElement cfg)
                 |> Dialog.withButtons buttons
                 |> dialogViewV2 cfg
             ]
-        , { defaultWithoutMenu | code = code }
+        , { defaultWithMenu | code = code }
         )
 
 
+dialogMobile : ExplorerStory
+dialogMobile =
+    story
+        ( "Dialog Mobile"
+        , Element.column
+            [ Element.height <| Element.px mobileResolution.height
+            , Element.width <| Element.px mobileResolution.width
+            , Element.centerX
+            ]
+            [ iconsSvgSprite
+            , Dialog.dialog demoText.title
```

Comment:
Can we use the same "Dialog" (everything until dialogViewV2 in a separated function) between stories?